docs/overview/software-design-document.mdx

```diff
@@ -80,7 +80,7 @@ A](#a---the-c4-model).
 import c4AerieContext from './assets/c4-aerie-context.png';
 
 <figure>
-  <img alt="Aerie Context" src={c4AerieContext} width="512" />
+  <img alt="Aerie Context" src={c4AerieContext} style={{ maxHeight: '100%', width: '700px' }} />
```

Comment:
The width on this picture was changed from 512px to 700px. Was this intentional?

Reply:
Yeah, it seemed a little small/hard to read at 512 so I upped it to 700 to match some of the others